admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,120 @@
+= Working with HTTP Proxies
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This should be a clear and concise description of the topic.
+:keywords: comma, separated, keywords, list
+
+toc::[]
+
+== Overview
+Production environments can deny direct access to the web and instead, have an HTTP or
+HTTPS proxy available. Configuring OpenShift builds and deployments to
+use these proxies is as simple as setting standard environment variables in
+configuration or JSON files.
+
+== Importing Image Streams
```

Comment:
@codificat Thanks for the tips. I see your points.

I've made the suggested changes, but kept the docker section as it's own (though closer to the above section) as from what I can tell it's still a different process for different purposes.

I received some pointers on some more info. I have to email some of the other OpenShift engineers. I'll include you on the email so that you can have some say in the process.

Thanks again.